demos/virtual_ai_assistant_demo/default_personality.yaml

```diff
@@ -24,3 +24,26 @@ summarize_the_user_prompt: >
   Strictly do not mention any personal data like age, name, gender, contact, non-health information etc. when summarizing.
   Summarize the health-related concerns mentioned by the patient in this conversation or in the provided context.
   You must include information from the context if it's provided.
+
+title:
```

Comment:
Let's change "title" to "instructions". This also requires changes in other files and README